tests/ng2web/src/app/room-service.service.spec.ts

```diff
@@ -151,6 +151,22 @@ describe('Service: Room Service', () => {
     })
   ));
 
+  it('should patch room attributes',
```

Comment:
Why is this change part of this pull request? it seems it squashed commits non related to this functionality.. This modifies the history